BUILD_LINUX.md

```diff
@@ -1,75 +1,96 @@
 # Build Linux
 
-*Last Updated on December 1, 2020*
+*Last Updated on December 1, 2021*
 
 Please read the [general build guide](BUILD.md) for information on dependencies required for all platforms. Only Linux specific instructions are found in this file.
 
 You can use the [Vircadia Builder](https://github.com/vircadia/vircadia-builder) to build on Linux more easily. Alternatively, you can follow the manual steps below.
 
-## Ubuntu 16.04/18.04 specific build guide
+## Ubuntu 16.04/18.04
+
 ### Ubuntu 16.04 only
-Add the following line to *.bash_profile*  
-`export QT_QPA_FONTDIR=/usr/share/fonts/truetype/dejavu/`
-### Ubuntu 18.04 server only
+
+Add the following line to `.bash_profile`:
+
+```bash
+export QT_QPA_FONTDIR=/usr/share/fonts/truetype/dejavu/
+```
+
+### Ubuntu 18.04 Server only
 Add the universe repository:  
-_(This is not enabled by default on the server edition)_
+_(This is not enabled by default on the server edition.)_
 ```bash
 sudo add-apt-repository universe
 sudo apt-get update
 ```
+
 #### Install build tools:
 -  First update the repositories:  
 ```bash
 sudo apt-get update -y
 sudo apt-get upgrade -y
 ```
--  git
+
+-  Install git
 ```bash
 sudo apt-get install git -y
 ```
-Verify by git --version  
--  g++
+Verify git was installed by running `git --version`.
+
+-  Install g++
 ```bash
 sudo apt-get install g++ -y
 ```
-Verify by g++ --version  
--  *Ubuntu 18.04* cmake
+Verify g++ was installed by running `g++ --version`.
+
+-  **Ubuntu 18.04** CMake
 ```bash
 sudo apt-get install cmake -y
 ```
-Verify by cmake --version  
-- *Ubuntu 16.04* cmake  
+Verify CMake was installed by running `cmake --version`.
+
+- **Ubuntu 16.04** CMake  
 ```bash
 wget https://cmake.org/files/v3.14/cmake-3.14.2-Linux-x86_64.sh
 sudo sh cmake-3.14.2-Linux-x86_64.sh --prefix=/usr/local --exclude-subdir
 ```
+
 #### Install build dependencies:
 -  OpenSSL:
 ```bash
 sudo apt-get install libssl-dev
 ```
-Verify with `openssl version`  
+Verify OpenSSL was installed by running `openssl version`.
+
 - OpenGL:
 ```bash
 sudo apt-get install libgl1-mesa-dev -y
 sudo ln -s /usr/lib/x86_64-linux-gnu/libGL.so.346.35 /usr/lib/x86_64-linux-gnu/libGL.so.1.2.0
```

Comment:
As far as I understand this is going to break a modern system.
Whatever the case, the system is already broken if this is necessary.
```suggestion
```